docs/source/Plugin/P036_commands.repl

```diff
@@ -42,14 +42,13 @@
         The line text can also be restored from the settings by the command ``restore``.
 
         All template notations can be used, like system variables, or reference to a task value.
-        They will be interpreted once to generate the new line content.
+        They will be interpreted once (while issuing the command) to generate the new line content.
 
         To use template notations that are to be reinterpreted each time before the line is displayed, the **<line> parameter must be negative**.
-        
-        Additionally to the **negative** <line> parameter following conditions must also be met:
 
-        * use ``$$<sysvar>$$`` instead of ``%<sysvar>%`` for system variables
-        * use ``${<TaskName>#<TaskValue>}$`` instead of ``[<TaskName>#<TaskValue>]`` for task values
+        Additionally to the **negative** <line> parameter following conditions must also be met:
+                * use ``\%<sysvar>\%`` instead of ``%<sysvar>%`` for system variables
+                * use ``\[<TaskName>#<TaskValue>\]`` instead of ``[<TaskName>#<TaskValue>]`` for task values
```

Comment:
You could also add an example for `\{some_function\}` to make the example exhaustive 😸

Reply:
Done!